modules/persistenceService/src/test/scala/migrations/PsqlMigrationsSpec.scala

```diff
@@ -0,0 +1,26 @@
+package migrations
+import cats.effect.IO
+import org.scalatest.funspec.AnyFunSpec
+import com.dimafeng.testcontainers.{ForAllTestContainer, PostgreSQLContainer}
+import config.{ApplicationConfig, ConnectionUrl, DriverName, PassWord, User}
+import org.scalatest.matchers.should.Matchers
+
+import java.sql.DriverManager
+
+class PsqlMigrationsSpec extends AnyFunSpec with ForAllTestContainer with Matchers {
+  override val container: PostgreSQLContainer = PostgreSQLContainer()
+
+  describe("Postgres container") {
+    it("should be started") {
+      val driverName    = DriverName(container.driverClassName)
+      val connectionUrl = ConnectionUrl(container.jdbcUrl)
+      val user          = User(container.username)
+      val password      = PassWord(container.password)
+
+      DbMigrations
+        .migrate[IO](ApplicationConfig(driverName, connectionUrl, user, password))
+        .map(_.migrationsExecuted shouldBe (1))
+        .unsafeRunSync()
```

Comment:
In a following PR we could use `cats-effect-testing-scalatest` and avoid the `unsafeRunSync`

https://github.com/typelevel/cats-effect-testing#scalatest

Reply:
I will amend it push a fix.
I guess `unsafe` is really unsafe.

Reply:
😂   
IMO we should have just one single `unsafe` per test. Your code is totally fine but using scalatest will allow us to be sure there's only one in the future.